src/events/mentionSpamExceeded.ts

```diff
@@ -17,14 +17,14 @@ export default class extends Event {
 		const lock = message.guild.moderation.createLock();
 		try {
 			await message.guild.members
-				.ban(message.author.id, { days: 0, reason: language.get(LanguageKeys.Monitors.NmsFooter) })
+				.ban(message.author.id, { days: 0, reason: t(LanguageKeys.Monitors.NoMentionSpamFooter) })
 				.catch((error) => this.client.emit(Events.ApiError, error));
-			await message
-				.sendLocale(LanguageKeys.Monitors.NmsMessage, [{ user: message.author }])
+			await message.channel
+				.sendTranslated(LanguageKeys.Monitors.NoMentionSpamMessage, [{ user: message.author }])
```

Comment:
Re-use t, also is user meant to be the mention or?
```suggestion
			await message.channel
				.send(t(LanguageKeys.Monitors.NoMentionSpamMessage, { user: message.author.toString() }))
```